charts/admission-controller/templates/webhook/admissionregistration.yaml

```diff
@@ -12,6 +12,7 @@ metadata:
   annotations:
     "helm.sh/hook": "post-install, post-upgrade"
 webhooks:
+{{- if .Values.scanner.enabled -}}
```

Comment:
The linter seems to be failing, could you try to remove the last `-}}` and leave it as `}}`?